src/test/java/com/iexec/common/docker/DockerRunRequestTests.java

```diff
@@ -51,16 +52,16 @@ void shouldAddSgxDeviceThenAddDevices() {
         devices.add(device2);
         DockerRunRequest request = DockerRunRequest.builder()
                 .containerName("containerName")
-                .isSgx(true) // notice order here
+                .sgxDriverMode(SgxDriverMode.LEGACY) // notice order here
```

Comment:
What of the other modes ? Should we test them as well ?

Reply:
Some new tests there: https://github.com/iExecBlockchainComputing/iexec-common/pull/308/commits/cd3695cb9161c2a152638e238c925d94d7e557d4 :)